presto-accumulo/pom.xml

```diff
@@ -369,4 +377,21 @@
             </build>
         </profile>
     </profiles>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.basepom.maven</groupId>
+                <artifactId>duplicate-finder-maven-plugin</artifactId>
+                <configuration>
+                    <ignoredResourcePatterns>
+                        <ignoredResourcePattern>org/apache/log4j/xml/log4j.dtd</ignoredResourcePattern>
+                    </ignoredResourcePatterns>
+                    <ignoredClassPatterns>
+                        <ignoredClassPattern>org.apache.log4j.*</ignoredClassPattern>
+                    </ignoredClassPatterns>
```

Comment:
We should not be ignoring duplicates. The duplicates need to be handled by excluding the dependencies